app/lib/constituency_api.rb

```diff
@@ -6,9 +6,25 @@ class Constituency < Struct.new(:name, :mp)
 
   class Mp < Struct.new(:id, :name, :start_date)
```

Comment:
Now we've overridden ``initialize`` there's not much we're getting from ``Struct`` - might be time to drop it and fill in the attr / eql stuff ourselves (as @pixeltrix suggested on the previous PR).